test/ProgressOnderwijsUtils.Tests/Data/PocoBulkCopyTest.cs

```diff
@@ -172,6 +184,70 @@ public void BulkCopySupportsCumputedColumn()
         PAssert.That(() => fromDb.Computed);
     }
 
+    [Fact]
+    public void BulkCopySupportsCumputedColumn_ViaInternalGetter()
+    {
+        var tableName = SQL($"#MyTable");
+        SQL(
+            $@"
+                create table {tableName} (
+                    Id int not null primary key
+                    , Computed as convert(bit, 1) -- deliberately not placed at the end
+                    , Bla nvarchar(max) not null
+                )
+            "
+        ).ExecuteNonQuery(Connection);
+
+        new[] {
+            new ComputedColumnViaInternalExample {
+                Id = 11,
+                Bla = "Something",
+            },
+        }.BulkCopyToSqlServer(Connection, BulkInsertTarget.LoadFromTable(Connection, tableName));
+
+        var fromDb = SQL(
+            $@"
+                select *
+                from {tableName}
+            "
+        ).ReadPocos<ComputedColumnExample>(Connection).Single();
+        PAssert.That(() => fromDb.Computed);
+    }
+
+    [Fact]
+    public void BulkCopySupportsCumputedColumn_WithoutSmallBatchOptimization()
+    {
+        var tableName = SQL($"#MyTable");
+        SQL(
+            $@"
+                create table {tableName} (
+                    Id int not null primary key
+                    , Computed as convert(bit, 1) -- deliberately not placed at the end
+                    , Bla nvarchar(max) not null
+                )
+            "
+        ).ExecuteNonQuery(Connection);
+
+        var asInserted = Enumerable.Range(0, 2000).Select(
+            id =>
+                new ComputedColumnExample {
+                    Id = id,
+                    Bla = "Something",
+                }
+        ).ToArray();
+        PAssert.That(() => asInserted.None(o => o.Computed));
+        asInserted.BulkCopyToSqlServer(Connection, BulkInsertTarget.LoadFromTable(Connection, tableName));
+
+        var fromDb = SQL(
+            $@"
+                select *
+                from {tableName}
+            "
+        ).ReadPocos<ComputedColumnExample>(Connection);
+        PAssert.That(() => fromDb.All(o => o.Computed));
+        PAssert.That(() => fromDb.Length == 2000);
+    }
+
     sealed record IncludingIdentityColumn : IWrittenImplicitly, IReadImplicitly
```

Comment:
Also assert error?

Reply:
assert error?

Reply:
Assert that writing on computed column fails